synfig-studio/src/gui/instance.cpp

```diff
@@ -1218,7 +1218,9 @@ Instance::make_param_menu(Gtk::Menu *menu,synfig::Canvas::Handle canvas, synfiga
 			param_list.add("selected_value_desc",selected_value_desc);
 	}
 
-	param_list.add("child",canvas_view->get_work_area()->get_active_bone_value_node());
+	if (std::string("bone") == canvas_view->get_smach().get_state_name()) {
+		param_list.add("child",canvas_view->get_work_area()->get_active_bone_value_node());
```

Comment:
what do you think about renaming `"child"` to `"active_bone"` ?

Reply:
This would require renaming this parameter in `valuedescbonesetparent.cpp` and maybe somewhere else, so I suggest leaving it as is for now.

Reply:
Only those two files.
It was the GSoC student that created it: just for ValueDescBoneSetParent action he created. 0948ac4b5883d4d10563a98fd0b38459ff7b6a38

Reply:
Done